### PR TITLE
Fixed category height overflow issue #4044

### PR DESCRIPTION
**Pull request by devansh-webkul**
**BUGS:**
Link: https://github.com/bagisto/bagisto/issues/4044

### CODE REVIEW

File: packages/Webkul/Category/src/Database/Seeders/CategoryBulkTableSeeder.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        CategoryRepository $categoryRepository
+    )
+    {
+        $this->faker = $faker;
+        $this->categoryRepository = $categoryRepository;
+    }
+
+    public function run()
+    {
+        for ($i=0; $i<$this->numberOfParentCategories; ++$i) {
```
Comment (jitendra-webkul) [inline packages/Webkul/Category/src/Database/Seeders/CategoryBulkTableSeeder.php:33]: Use space before and after with binary operator

File: packages/Webkul/Category/src/Database/Seeders/CategoryBulkTableSeeder.php [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                'slug'        => $this->faker->slug,
+                'name'        => $this->faker->firstName,
+                'description' => $this->faker->text(),
+                'parent_id'   => 1,
+                'status'      => 1,
+            ]);
+
+            if ($createdCategory) {
+
+                for ($j=0; $j<$this->numberOfChildCategories; ++$j) {
```
Comment (jitendra-webkul) [inline packages/Webkul/Category/src/Database/Seeders/CategoryBulkTableSeeder.php:45]: Use space before and after with binary operator

File: packages/Webkul/Category/src/Database/Seeders/CategoryBulkTableSeeder.php [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    )
+    {
+        $this->faker = $faker;
+        $this->categoryRepository = $categoryRepository;
+    }
+
+    public function run()
+    {
+        for ($i=0; $i<$this->numberOfParentCategories; ++$i) {
+
```
Comment (jitendra-webkul) [inline packages/Webkul/Category/src/Database/Seeders/CategoryBulkTableSeeder.php:34]: Remove new line

File: packages/Webkul/Category/src/Database/Seeders/CategoryBulkTableSeeder.php [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+            $createdCategory = $this->categoryRepository->create([
+                'slug'        => $this->faker->slug,
+                'name'        => $this->faker->firstName,
+                'description' => $this->faker->text(),
+                'parent_id'   => 1,
+                'status'      => 1,
+            ]);
+
+            if ($createdCategory) {
+
```
Comment (jitendra-webkul) [inline packages/Webkul/Category/src/Database/Seeders/CategoryBulkTableSeeder.php:44]: Remove new line

File: packages/Webkul/Category/src/Database/Seeders/CategoryBulkTableSeeder.php (hunk 1 above)
Comment (devansh-webkul) [reply packages/Webkul/Category/src/Database/Seeders/CategoryBulkTableSeeder.php]: Done

File: packages/Webkul/Category/src/Database/Seeders/CategoryBulkTableSeeder.php (hunk 2 above)
Comment (devansh-webkul) [reply packages/Webkul/Category/src/Database/Seeders/CategoryBulkTableSeeder.php]: Done

File: packages/Webkul/Category/src/Database/Seeders/CategoryBulkTableSeeder.php (hunk 3 above)
Comment (devansh-webkul) [reply packages/Webkul/Category/src/Database/Seeders/CategoryBulkTableSeeder.php]: Done

**Pull request merged**